src/components/download/DownloadDialog.js

```diff
@@ -78,57 +64,64 @@ export class DownloadDialog extends Component {
         const isSupported = downloadSupport() && !this.state.error;
 
         return (
-            <Dialog open={showDialog} onClose={this.onClose}>
-                <DialogTitle disableTypography={true}>
-                    {i18n.t('Download map')}
-                </DialogTitle>
-                <DialogContent className={classes.content}>
-                    {isSupported ? (
-                        <Fragment>
-                            <div className={classes.checkbox}>
-                                <Checkbox
-                                    label={i18n.t('Show name')}
-                                    checked={showName}
-                                    disabled={!hasName}
-                                    onCheck={toggleDownloadShowName}
-                                />
-                            </div>
-                            <div className={classes.checkbox}>
-                                <Checkbox
-                                    label={i18n.t('Show legend')}
-                                    checked={showLegend}
-                                    disabled={!hasLegend}
-                                    onCheck={toggleDownloadShowLegend}
-                                />
-                            </div>
-                            {hasLegend && showLegend && (
-                                <LegendPosition
-                                    position={legendPosition}
-                                    onChange={setDownloadLegendPosition}
-                                />
-                            )}
-                        </Fragment>
-                    ) : (
-                        i18n.t(
-                            'Map download is not supported by your browser. Try Google Chrome or Firefox.'
-                        )
-                    )}
-                </DialogContent>
-                <DialogActions>
-                    <Button color="primary" onClick={this.onClose}>
-                        {isSupported ? i18n.t('Cancel') : i18n.t('Close')}
-                    </Button>
-                    {isSupported && (
-                        <Button
-                            variant="contained"
-                            color="primary"
-                            onClick={this.onDownload}
-                        >
-                            {i18n.t('Download')}
+            <Modal
+                open={showDialog}
```

Comment:
I don't think `Modal` has an `open` prop, usually we use conditional rendering instead

Reply:
Fixed.